quora-service/src/main/java/com/upgrad/quora/service/entity/UserEntity.java

```diff
@@ -68,6 +70,15 @@
     @NotNull
     private String mobile;
 
+    @OneToMany(mappedBy = "user_id", cascade = CascadeType.REMOVE)
+    List<AnswerEntity> answers = new ArrayList<>();
```

Comment:
Here you need not create a new object, List<AnswerEntity> answers; 
this is enough